stack.yaml

```diff
@@ -51,7 +51,7 @@ extra-deps:
   - chronos-1.1.5
   - co-log-0.5.0.0
   - decision-diagrams-0.2.0.0
-  - typerep-map-0.6.0.0
+  - typerep-map-0.5.0.0
```

Comment:
I did not see a good reason to use version 0.6 here and it created problems for cabal.
_Not_ going to delete `allow-newer` for now, though...